src/Classes/Controllers/LoginController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Portal\Controllers;
+
+use \Slim\Http\Request as Request;
+use \Slim\Http\Response as Response;
+use Portal\Models\UserModel;
+
+class LoginController
+{
+    private $userModel;
+
+    /**
+     * Instantiates login controller
+     *
+     * LoginController constructor.
+     * @param UserModel $userModel userModel object dependency
+     */
+    public function __construct(UserModel $userModel)
+    {
+        $this->userModel = $userModel;
+    }
+
+    /**
+     * Verifies user credentials and returns message and status code
+     *
+     * @param Request $request HTTP request
+     * @param Response $response HTTP response
+     *
+     * @return string JSON message and status code
+     */
+    public function __invoke(Request $request, Response $response)
+    {
+        $data = ['success' => false, 'msg' => 'Incorrect email or password.', 'data' => []];
+        $statusCode = 401;
+        $_SESSION['loggedIn'] = false;
+
+        $parsedBody = $request->getParsedBody();
+        $user = $this->userModel->getUserByEmail($parsedBody['userEmail']);
```

Comment:
email validation?

Reply:
fetches saved data to compare with form input.